### PR TITLE
Fix pkg/controller typos in some error messages, comments etc

### PR DESCRIPTION
**Pull request by taesunny**
**What type of PR is this?**
> /kind cleanup

**What this PR does / why we need it**:
> To fix pkg/controller Typos in comments, error messages

**Which issue(s) this PR fixes**:

> Fixes #

**Special notes for your reviewer**:
> NONE

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:
> NONE

### CODE REVIEW

**Comment by taesunny** [comment]
I make an assignment from k8s-ci-robot APPROVALNOTIFIER info.

/assign @derekwaynecarr

File: pkg/controller/daemon/daemon_controller_test.go [hunk 1]
```diff
@@ -537,7 +537,7 @@ func TestSimpleDaemonSetScheduleDaemonSetPodsLaunchesPods(t *testing.T) {
 		}
 
 		if len(nodeMap) != 0 {
-			t.Fatalf("did not foud pods on nodes %+v", nodeMap)
+			t.Fatalf("did not found pods on nodes %+v", nodeMap)
```
Comment (sanchezl) [inline pkg/controller/daemon/daemon_controller_test.go]:
```suggestion
			t.Fatalf("did not find pods on nodes %+v", nodeMap)
```

File: pkg/controller/replicaset/replica_set_test.go [hunk 2]
```diff
@@ -1196,7 +1196,7 @@ func TestExpectationsOnRecreate(t *testing.T) {
 		t.Errorf("No expectations found for ReplicaSet %q", oldRSKey)
 	}
 	if rsExp.Fulfilled() {
-		t.Errorf("There should be unfulfiled expectation for creating new pods for ReplicaSet %q", oldRSKey)
+		t.Errorf("There should be unfulfilled expectation for creating new pods for ReplicaSet %q", oldRSKey)
```
Comment (sanchezl) [inline pkg/controller/replicaset/replica_set_test.go]:
```suggestion
		t.Errorf("There should be unfulfilled expectations for creating new pods for ReplicaSet %q", oldRSKey)
```

File: pkg/controller/replicaset/replica_set_test.go [hunk 3]
```diff
@@ -1275,7 +1275,7 @@ func TestExpectationsOnRecreate(t *testing.T) {
...
```
Comment (sanchezl) [inline pkg/controller/replicaset/replica_set_test.go]:
```suggestion
		t.Errorf("There should be unfulfilled expectations for creating new pods for ReplicaSet %q", oldRSKey)
```

**Review by sanchezl — CHANGES_REQUESTED** [review]

**Comment by taesunny** [comment]
Thank you so much for your review!!
I add a commit applying your suggestions.
I'll squash two commits!
@sanchezl

**Comment by taesunny** [comment]
I squashed a commit applying review result into the original commit.
And I pushed.

**Review by sanchezl — APPROVED** [review]
/lgtm

**Pull request merged**